airflow/example_dags/plugins/event_listener.py

```diff
@@ -164,9 +164,10 @@ def on_dag_run_running(dag_run: DagRun, msg: str):
     """
     print("Dag run  in running state")
     queued_at = dag_run.queued_at
-    dag_hash_info = dag_run.dag_hash
 
-    print(f"Dag information Queued at: {queued_at} hash info: {dag_hash_info}")
+    dag_version = dag_run.dag_version.version if dag_run.dag_version else None
```

Comment:
probably this could be made cleaner py putting an association proxy on dag run, or perhaps just a property. it's just a bit clunky to have to do the if else here.

Reply:
Since it's a foreignkey ref, I decided to ensure the dag_version is always available by providing it in unittest. In normal run, the dag_version will always be available.